README.rst

```diff
@@ -46,12 +46,12 @@ As a summary:
 Install
 *******
 
-Currently developped against Python 2.7.x.
-Supports only Git for now.
+Currently developped to support Python 2.7 and Python 3.4+.
+Supports Git only for now.
 
 .. code-block:: sh
 
-   sudo pip install https://github.com/benoitbryon/psykorebase/archive/master.tar.gz#egg=psykorebase
+   sudo pip install psykorebase
```

Comment:
But perhaps "sudo" is enough, as it is a straightforward way to install the software. Then users who master pip will guess how to manage alternate installs.